Stack/Opc.Ua.Core/Types/Encoders/EncodeableFactory.cs

```diff
@@ -380,6 +380,9 @@ public void AddEncodeableTypes(Assembly assembly)
 
                         AddEncodeableType(systemTypes[ii]);
                     }
+
+                    // only needed while adding assembly types
+                    m_unboundTypeIds.Clear();
```

Comment:
m_unboundedTypeIds is only used in this method and AddEncodeable, should be local variable instead of class member

Reply:
refactored to use only local dictinary